plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy_test.go

```diff
@@ -91,3 +92,15 @@ func TestControllerRoleLabel(t *testing.T) {
 		}
 	}
 }
+
+func TestControllerRoleVerbsConsistensy(t *testing.T) {
+	roles := ControllerRoles()
```

Comment:
should we also check NamespaceRoles(), NodeRules(), and ClusterRoles()?

Reply:
Added.